frontend/subrequests/lint/lint.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+type SourceUpdateCallback func(*pb.SourceInfo) *pb.SourceInfo
```

Comment:
nit: `SourceInfoMap` (or just `SourceMap`) could be an ok name for this, I think.

Reply:
Updated!